packages/common/src/store/account/sagas.ts

```diff
@@ -107,23 +107,24 @@ export function* fetchAccountAsync({ isSignUp = false }): SagaIterator {
   yield* put(signedIn({ account, isSignUp }))
 }
 
-export function* cacheAccount(account: User) {
+export function* cacheAccount(account: AccountUserMetadata) {
+  const { user: accountUser } = account
   const localStorage = yield* getContext('localStorage')
 
   yield put(
     cacheActions.add(Kind.USERS, [
-      { id: account.user_id, uid: 'USER_ACCOUNT', metadata: account }
+      { id: accountUser.user_id, uid: 'USER_ACCOUNT', metadata: account }
     ])
   )
-
+  const { playlists: collections, ...user } = accountUser
```

Comment:
Need to update [`fetchAccountLocalAsync`](https://github.com/AudiusProject/audius-protocol/pull/10447/commits/b30795bdee9a59e01f586a089c484293d1fcfc94#diff-14c98feff24bb8d627b6e0e5a2d3900eaef4d58b57c030d6a06c188577defc2aR134) and anything else that calls into this function to pass the right shape.